swagger.yml

```diff
@@ -1022,35 +1043,48 @@ components:
         Microsoft/SQLServer:
           type: boolean
           nullable: true
+        example-1: {}
     TechnologyStatus:
       type: object
-      required:
-        - Product
-        - HostsCount
-        - Compliance
-        - UnpaidDues
       properties:
         Product:
           type: string
+          description: technologial product code
```

Comment:
I don't know what to say...
@cirelli94 what do you say about "technologial product code"?